core/metadata.js

```diff
@@ -360,10 +360,14 @@ const makeComparator = (name, interestingFields) => {
       })
     })
   }
+  const isRealDiff = (difference) => {
+    const diff = difference.item ? difference.item : difference
```

Comment:
[nitpick] You prefer the ternary to `const diff = difference.item || difference`?

Reply:
It's a reflex and I was using `!!difference.item` in the condition until the linter complained.
I didn't think about changing the way I was writing this.

Your solution is shorter and can be implemented.